g11n-ws/modules/md-service-i18n-l2/src/main/java/com/vmware/i18n/l2/service/pattern/PatternServiceImpl.java

```diff
@@ -175,6 +169,19 @@
 		return patternMap;
 	}
 
+	private void handleSpecialCategory(String category, String language, Map<String, Object> categoriesMap) throws VIPCacheException {
+		Map<String, Object> pluralPatternMap = getPattern(language, Arrays.asList(category));
```

Comment:
The variable name 'pluralPatternMap' need be changed, since it is changed according to category.

Reply:
fixed